src/main/resources/project.properties

```diff
@@ -1 +1 @@
-firebaseApiKey=AIzaSyDO-JXjcrO9x5sSX30mLQdSpu3_r7yI9gY
\ No newline at end of file
+firebaseApiKey=AIzaSyDwAIdHtTi94JiG4NDhyRgXrz3dk3RIT8E
```

Comment:
@micklynch you forgot to replace with the staging key

Reply:
Sorry @carlosfaria94 , changed.